ui/component/viewers/videoViewer/internal/videojs.jsx

```diff
@@ -209,6 +226,13 @@ export default React.memo<Props>(function VideoJs(props: Props) {
       // Initialize mobile UI.
       player.mobileUi();
 
+      if (!embedded) {
+        window.player.bigPlayButton && window.player.bigPlayButton.hide();
+      } else {
+        const bigPlayButton = document.querySelector('.vjs-big-play-button');
+        if (bigPlayButton) bigPlayButton.style.setProperty('display', 'block', 'important');
```

Comment:
_Not a blocker._
This is also being set in css down below.  Just curious if both is needed or just pending cleanup.
Also spoke with Jessop and he agrees that DOM manipulation is bad/weird in React (for various reasons), but perhaps there's no other choice.

```
// TODO: make sure there's no bad side effects of this
button.vjs-big-play-button {
  display: none !important;
}
```